### ISSUE TITLE
prediction fails

### ISSUE THREAD

**Issue by Motorrat** [automl/auto-sklearn#246]
```
    y_pred = automl.predict(X_unknown.values)
  File "/home/ubuntu/anaconda3/lib/python3.5/site-packages/autosklearn/estimators.py", line 373, in predict
    return super(AutoSklearnClassifier, self).predict(X)
  File "/home/ubuntu/anaconda3/lib/python3.5/site-packages/autosklearn/estimators.py", line 66, in predict
    return self._automl.predict(X)
  File "/home/ubuntu/anaconda3/lib/python3.5/site-packages/autosklearn/estimators.py", line 532, in predict
    predicted_probabilities = self._automl.predict(X)
  File "/home/ubuntu/anaconda3/lib/python3.5/site-packages/autosklearn/automl.py", line 505, in predict
    prediction = model.predict_proba(X_)
  File "/home/ubuntu/anaconda3/lib/python3.5/site-packages/autosklearn/pipeline/classification.py", line 127, in predict_proba
    return self.steps[-1][-1].predict_proba(Xt)
  File "/home/ubuntu/anaconda3/lib/python3.5/site-packages/autosklearn/pipeline/components/classification/__init__.py", line 126, in predict_proba
    return self.choice.predict_proba(X)
  File "/home/ubuntu/anaconda3/lib/python3.5/site-packages/autosklearn/pipeline/components/classification/random_forest.py", line 107, in predict_proba
    probas = self.estimator.predict_proba(X)
  File "/home/ubuntu/anaconda3/lib/python3.5/site-packages/sklearn/ensemble/forest.py", line 537, in predict_proba
    X = self._validate_X_predict(X)
  File "/home/ubuntu/anaconda3/lib/python3.5/site-packages/sklearn/ensemble/forest.py", line 319, in _validate_X_predict
    return self.estimators_[0]._validate_X_predict(X, check_input=True)
  File "/home/ubuntu/anaconda3/lib/python3.5/site-packages/sklearn/tree/tree.py", line 376, in _validate_X_predict
    % (self.n_features_, n_features))
ValueError: Number of features of the model must  match the input. Model n_features is 15 and  input n_features is 14

```

**Comment by mfeurer** [comment]
Can you provide some code which allows me to reproduce this issue? Also, how often does this issue appear?

**Comment by Motorrat** [comment]
it may be the same issue as https://github.com/automl/auto-sklearn/issues/263 where I have sent you the steps and the dataset to reproduce.

**Comment by mfeurer** [comment]
@Motorrat could you please check whether this issue still exists in auto-sklearn 0.2.0?

**Comment by mfeurer** [comment]
I just released auto-sklearn 0.3.0. Could you please check if the issue is still there, and if yes, reopen this issue?
